main/ZgatewayBT.ino

```diff
@@ -95,7 +97,16 @@ void pubBTMainCore(JsonObject& data, bool haPresenceEnabled = true) {
   if (abs((int)data["rssi"] | 0) < minRssi && data.containsKey("id")) {
     String mac_address = data["id"].as<const char*>();
     mac_address.replace(":", "");
+#  ifdef MQTTDecodeTopic
+    String mactopic;
+    if (data.containsKey("model")) {
+      mactopic = subjectBTtoMQTT + String("/") + mac_address;
+    } else {
+      mactopic = String("/") + MQTTDecodeTopic;
```

Comment:
Is it intended that `mactopic` begins with `/` or is `subjectBTtoMQTT` is missing ?
(I'm reviewing this code to merge PR #1220 with it and this seems strange)

Reply:
Hi @BadWolf42, for the new undecoded messages isn't it then **/** + **MQTTDecodeTopic**, which in this case is

`'-DMQTTDecodeTopic="BTtoMQTT/undecoded"'`

-> '/BTtoMQTT/undecoded'

Reply:
Ok, so you confirm that L105 is OK and should **NOT** be:
`mactopic = subjectBTtoMQTT + String("/") + MQTTDecodeTopic;` 
to publish in `<GW_NAME>/BTtoMQTT/undecoded` with `'-DMQTTDecodeTopic="undecoded"'`
Right?

(Forgive me but a heading / is not very frequent in mqtt)

Reply:
I'll better leave the confirmation to @h2zero or @1technophile why it was decided to define it this way ;) and yes, your variation

mactopic = subjectBTtoMQTT + String("/") + MQTTDecodeTopic;

would also be valid with MQTTDecodeTopic just being

```
// Uncomment to send undecoded device data to another gateway device for decoding
// #define MQTTDecodeTopic    "undecoded"
```

`'-DMQTTDecodeTopic="undecoded"'`

> (Forgive me but a heading `/` is not very frequent in mqtt)

but either way, even **subjectBTtoMQTT** has a heading `/`, and in the end `mactopic` will always be a trailing substring after `home/OpenMQTTGateway_ESP32_BLE`

I'm very much looking forward to your iBeacon UUID PR as well, but assume that since it also won't decoder if the flag is set, a similar implementation would also be needed in the receiving Theengs Gateway end when it receives the raw manufacturerdata.

Reply:
I think L105 should be :
`mactopic = subjectBTtoMQTT + String("/") + MQTTDecodeTopic;`

And L56 of config_BT.h should be:
`// #define MQTTDecodeTopic    "undecode"`

Reply:
Ok thank you both for the confirmation, I'll update my PR accordingly 👍